README.md

```diff
@@ -18,70 +18,53 @@ may well be used to fire requests to other services.
-    * ```bundle exec rspec spec```
 
-* Services (job queues, cache servers, search engines, etc.)
-    
+## Initial setup
 
-* Deployment instructions
-    * check the code out and run ```rails s```
+Run ```make initial-setup```
 
-* play with application
-    Once the server is started you can actualy use postman to fire requests using the endpoints below
-    
+This will create the db schema  and run a  migration. you shouldn't have to run this again.
 
-* Developer local Endpoints
-    * ``http://localhost:3000/api/v1/applications``
-        * Only POST is supported at the moment so this wil create an application and retrun application ref
-    * ``http://localhost:3000/api/v1/status``
-        * Only GET is supported at the moment not sure anything else is needed here
+## Running the tests
 
+In order to run the tests run
 
-## Docker
+ * ```make test```
+ 
+ This is currently setup to run all rspec tests
 
-The docker file is created against a ruby alpine image using version 2.5.1 which is consistent
-with the latest version at the time of writing.
+## Running the application
 
-Alpine images are usually very lightweight and such are preferred choice for deploying containers.
+ * ```make serve```
+ 
+ This will use docker compose to start postgres and the api project.
+ requests will be served on port localhost:3000
+ 
 
-In order to create a local build you can run
+## Other options
 
-```docker build -t laa-apply-for-legalaid-api .```
+There are other configurations in the makefile which you can use if you want 
 
+i.e. ```start-local-server```
 
-And then run the image using
+This will start a server without docker and setup a postgres db exposed at localhost:5432.  
+Benefit of this is you dont have to build container everytime you make a change to a file.  handy for the html/css changes.
 
-```docker container run  -d -p 3000:3000 laa-apply-for-legalaid-api```
 
 
-## Postgres Sql
+## Developer local Endpoints
 
-If you already have an installation of postgres you might want to use it.
-For development mode i am going for the simplest setup without setting any specific roles or users
+* Post an application 
 
-If you dont have postgres installed then you can run the following command to get up and running quickly
+```http://localhost:3000/api/v1/applications```    
 
-```docker run  -d -v pg-data:/var/lib/postgresql/data -p 5432:5432  --name postgres  postgres:10.5```
- 
- the 10.5 version appears to be the latest and most likely the one we will be using in RDS.
- 
- The above will start a postgres server and expose it on ```localhost:5432```
- 
- 
- If you are running this for the first time you will have to run the following commands form within the project
- 
- ```rake db:setup```
- 
- ```rake db:migrate```
- 
- 
-For every other time you do a pull it might be a good idea to 
-```rake db:migrate```
+* Get status of the service 
 
-It's also good idea at this pint to rerun the tests.
+```http://localhost:3000/api/v1/status```
```

Comment:
I think we need to be careful about running everything on 3000 - it will conflict with our other apps. Can we change this to 3002?

That will meam
Provider frontend = 3000
Citizen frontend = 3001
API = 3002

Reply:
`/api/v1/status` should be `/v1/status`